internal/business/protect/protect.go

```diff
@@ -69,7 +73,9 @@ func (p *GraphQLProtect) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 }
 
 func (p *GraphQLProtect) handle(w http.ResponseWriter, r *http.Request) {
-	errs := p.validateRequest(r)
+	payloads, errs := p.validateRequest(r)
+
+	p.accessLogging.Log(payloads, r.Header)
```

Comment:
when there is a validation error payloads is nil, should we still log then? or should the error also be passed to the log?

Reply:
The `forr` will not log anything if `payloads` is `nil`.  Good point about logging `errs`, that might prove valuable. It might also produce a ton of logs when validations fail, which we might not be interested in. 

Would you agree this is something we can postpone to see if there is value in logging the errs?

Reply:
Yeah but I dont know that log will deal with nil's just fine, my first thought when reviewing this code is "hmm I wonder if that error should be checked first?". Im fine keeping it like this, just something that stood out to me while reviewing